models/discordactions.js

```diff
@@ -150,5 +204,6 @@ module.exports = {
   addGroupRoleToMember,
   isGroupRoleExists,
   updateDiscordImageForVerification,
```

Comment:
+1

Reply:
Did not understand the comment here? 
Can you please elaborate?